<?xml version="1.0" encoding="UTF-8"?>

<!--
******************************************************************
 ** @copyright Copyright (c) 2012 PROFACTOR GmbH. All rights reserved. 
 **
 ** Redistribution and use in source and binary forms, with or without
 ** modification, are permitted provided that the following conditions are
 ** met:
 **
 **     * Redistributions of source code must retain the above copyright
 ** notice, this list of conditions and the following disclaimer.
 **     * Redistributions in binary form must reproduce the above
 ** copyright notice, this list of conditions and the following disclaimer
 ** in the documentation and/or other materials provided with the
 ** distribution.
 **     * Neither the name of Google Inc. nor the names of its
 ** contributors may be used to endorse or promote products derived from
 ** this software without specific prior written permission.
 **
 ** THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
 ** "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
 ** LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
 ** A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
 ** OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
 ** SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
 ** LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
 ** DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
 ** THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 ** (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 ** OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 **
 ** @authors christoph.kopf@profactor.at
 **          florian.eckerstorfer@profactor.at
******************************************************************
-->

<ui version="4.0">
 <class>hardware_key_dialog</class>
 <widget class="QWidget" name="hardware_key_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReconstructMe Hardware Hashes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>The anonymized hardware hashes below are used by the ReconstructMe team
to generate licenses for individual computers. Please send them in as advised when
requesting a single user license.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="hw_key_te">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>8</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cp_clipboard_btn">
         <property name="text">
          <string>Copy to clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
